### ISSUE TITLE
EULA and Registration Preventing Run

### ISSUE THREAD

**Issue by macitpros** [acodega/mdmNotify#1]
I noticed that when testing the script, it wouldn't run. The logs mentioned EULA and registration was blank. Commenting out the lines fixed the issue and was able to proceed with testing.

There doesn't appear to be a spot where the variables `$REGISTRATION_ENABLED` and `$EULA_ENABLED` is defined, but I may just be missing it.

Perhaps an if statement when in `testing` mode to skip the the lines below?

https://github.com/acodega/mdmNotify/blob/beb587361787e079d189b83d746cec0b20259c17/mdmNotify.sh#L164-L171

**Comment by acodega** [comment]
Good catch, I overlooked that these checks were still enabled. I had taken out the variables themselves because it doesn't help much using EULA and registration with Mosyle. We should readd the values and default as false. (User registration would need to be designed for Mosyle, the script would need to save the information locally, and then you ned to add a custom attribute to read the information and save it to Mosyle. Keep in mind if you want to make device groups off of those values, those groups only calculate membership every 24 hours, so they would be slow to realize new members.)

**Comment by acodega** [comment]
So I'm wondering wether to keep this idea going. [Dialog released 1.9.1 today](https://github.com/bartreardon/Dialog) which supports [Lists](https://github.com/bartreardon/Dialog-scripts/tree/main/Lists), and even has a sample script using Installomator. Lists are a dialog box that displays to the user a list of apps like a checklist with progress bar. You can also use dialog for things like EULA and registration, albeit not inside one window the way DEP Notify does. My goal was always more about providing user feedback while initial deployment ran rather than collecting data like registration data, since there wasn't a great way to get that info back to Mosyle quickly.
